handlers/namespaces.go

```diff
@@ -33,9 +33,17 @@ func NamespaceList(w http.ResponseWriter, r *http.Request) {
 // NamespaceValidationSummary is the API handler to fetch validations summary to be displayed.
 // It is related to all the Istio Objects within the namespace
 func NamespaceValidationSummary(w http.ResponseWriter, r *http.Request) {
+	query := r.URL.Query()
 	vars := mux.Vars(r)
 	namespace := vars["namespace"]
 
+	cluster := ""
+	if query.Has("cluster") && query.Get("cluster") != "" {
```

Comment:
I think you can simplify these checks to just be `query.Get("cluster") != ""`. `cluster==""` means the same thing as not providing cluster as a query param. Technically I think `""` could be a valid name for a cluster but if that's true then you should just check for `query.Has("cluster")`.